pontoon/contributors/templates/contributors/includes/timeline.html

```diff
@@ -23,7 +23,7 @@
         {% if event.translation %}
         {% set direction = event.translation.locale.direction %}
         <div class="fa fa-2x quote {% if direction == 'ltr' %}fa-quote-left{% else %}fa-quote-right{% endif %}"></div>
-        <p dir="{{ direction }}" lang="{{ event.translation.locale.code }}">{{ event.translation.string }}</p>
+        <p dir="{{ direction }}" lang="{{ event.translation.locale.code }}" data-script="{{ event.translation.locale.script }}">{{ event.translation.string }}</p>
```

Comment:
I wonder if backend could prefetch this script - that could reduce a number of queries.